examples/000_Local_Deployment.ipynb

```diff
@@ -0,0 +1,352 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Local Deployment\n",
+    "\n",
+    "This example steps through how to deploy a local version of DTBase for development purposes. These instructions are a simplication of the developer docs and more details can be found there. \n",
+    "\n",
+    "There are two options for deploying DTBase locally:\n",
+    "\n",
+    "1. Docker Compose \n",
+    "2. Command Line \n",
+    "\n",
+    "Although docker compose is easier, it is faster to develop via deployment from the command line. This is because the docker images have to be rebuilt everytime a change is made to the code. The rest of this example will walk through the local deployment of DTBase via the command line. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Running DTBase Locally"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fork Repository and Install DTBase"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- Fork the DTBase repository: https://github.com/alan-turing-institute/DTBase and clone a copy to your local machine.  \n",
+    "- Navigate into the repository on the command line and install the dtbase package and dependencies\n",
+    "\n",
+    "`pip install -e .[dev]`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set Environment Variables\n",
+    "\n",
+    "Located in the .secrets folder, rename `dtenv_template.sh` to `dtenv_localdb.sh` (or any other name) and populate the variables. There is more information of what these values mean in the developer docs. The below is the minimal example needed to run dtbase locally. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "bat"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "export DT_SQL_TESTUSER=\"postgres\"\n",
+    "export DT_SQL_TESTPASS=\"password\"\n",
+    "export DT_SQL_TESTHOST=\"localhost\"\n",
+    "export DT_SQL_TESTPORT=\"5432\"\n",
+    "export DT_SQL_TESTDBNAME=\"test_db\"\n",
+    "\n",
+    "export DT_SQL_USER=\"postgres\"\n",
+    "export DT_SQL_PASS=\"database_password\"\n",
+    "export DT_SQL_HOST=\"localhost\"\n",
+    "export DT_SQL_PORT=\"5432\"\n",
+    "export DT_SQL_DBNAME=\"dt_dev\"\n",
+    "\n",
+    "export DT_BACKEND_URL=\"http://localhost:5000\"\n",
+    "export DT_DEFAULT_USER_PASS=\"user_password\"\n",
+    "export DT_FRONT_SECRET_KEY=\"frontend_password\"\n",
```

Comment:
Could we call these two `frontend/backden_secret_key` instead of `password`? Makes it clearer that isn't anything anyone ever needs to enter into a password field. Alternatively we could leave these as `<REPLACE ME>` like in the dev docs, to make it clear that the user should come up with their own values.